backend/Origam.Security.Identity/IdentityServiceAgent.cs

```diff
@@ -773,6 +780,34 @@ private XmlDocument GetUserDataXml(IOrigamUser user)
             xmlDoc.AppendChild(root);
             return xmlDoc;
         }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposedValue)
+            {
+                if (disposing)
+                {
+                    serviceScope?.Dispose();
+                }
+                // TODO: free unmanaged resources (unmanaged objects) and override finalizer
+                // TODO: set large fields to null
+                disposedValue = true;
+            }
+        }
+
+        // // TODO: override finalizer only if 'Dispose(bool disposing)' has code to free unmanaged resources
+        // ~IdentityServiceAgent()
+        // {
+        //     // Do not change this code. Put cleanup code in 'Dispose(bool disposing)' method
+        //     Dispose(disposing: false);
+        // }
+
+        public void Dispose()
+        {
+            // Do not change this code. Put cleanup code in 'Dispose(bool disposing)' method
+            Dispose(disposing: true);
+            GC.SuppressFinalize(this);
+        }
     }

```

Comment:
This class does not have any unmanaged resources. So according to: https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#cascade-dispose-calls
We can just implement the "void Dispose()" method and make the class sealed. I would also suggest adding a comment explaining why the class is sealed.